content/rule_book/roles_and_responsibilities.md

```diff
@@ -19,22 +19,22 @@ Teams can have up to 5 quizzers on it. Teams can have up to 4 quizzers in the qu
 - No comment other than "correct" or "incorrect" need be announced by the quiz officials in making a decision.
     - However, at the discretion of the answer judge and quizmaster, or at the request of the captain, additional information may be given:
         - If the additional information regarding the ruling is provided at the discretion of the AJ/QM, and not at the request of the captain, this is not considered an explanation of a ruling (which would invalidate a Challenge or Protest).
-        - If the captain requests additional information regarding the ruling, this is considered an explanation of a ruling, and invalidates a Challenge or Protest. Before replying to a request for explanation, the QM must ask all teams if they wish to Challenge.
+        - If the captain requests additional information regarding the ruling, this is considered an explanation of a ruling and invalidates a Challenge or Protest. Before replying to a request for an explanation, the QM must ask all teams if they wish to Challenge.
 
 ### Quizmasters (QM)
 
 The Quizmaster must:
 
 - Identify the question type of each question and begin every question with, "question number_____, question".
 - Rule immediately upon an obviously correct or incorrect answer if there is no indication from the answer judge to the contrary.
-    - When an answer is in question (different than what is on the card), the quizmaster will discuss with the answer judge.
+    - When an answer is in question (different than what is on the card), the quizmaster will discuss it with the answer judge.
 - During the 30-second answering period use the following prompts:
     - If a quizzer has provided all the information in the answer, except the identification of a pronoun present which is part of the answer text, the quizmaster will ask the quizzer to identify the specific pronoun, (example: "Can you identify 'He'?" or "Can you identify 'They'?") The quizmaster is not allowed to say anything other than what the quizzer has already said when asking for the identification. A pronoun may need to be identified only if the identification (antecedent) is in context.
-        - Once the quizzer has completed all information in the answer, the QM will prompt for the quizzer to identify the specific pronoun, even if the quizzer has not yet completed all information in the question.
+        - Once the quizzer has completed all information in the answer, the quizmaster will prompt the quizzer to identify the specific pronoun, even if the quizzer has not yet completed all information in the question.
```

Comment:
Gryphon can decide in future OCD passes, but I was trying to adhere to AP style here. Quizmaster is a defined term with a defined abbreviation.

Reply:
I changed it as it was the only use of QM in that document section.

Reply:
It's all good. If you catch term things like this, feel free to edit, but also feel equally free to ignore them for now. I'll catch them in phase 2. Worrying about global term style unity right now is a bit of wasted effort, I think.